app/src/main/java/de/ph1b/audiobook/features/bookOverview/EditBookBottomSheet.kt

```diff
@@ -58,8 +58,9 @@ class EditBookBottomSheet : BottomSheetDialogFragment() {
     dialog.setContentView(container.containerView)
 
     container.title.setOnClickListener {
-      EditBookTitleDialogFragment.newInstance(book)
```

Comment:
Can you replace the `EditBookBottomSheet` with a `DialogController` too? It seems that `BottomSheetDialogFragment` just wraps the dialog in an additional `BottomSheetDialog` so we might just omit that.